pkgs/applications/blockchains/zcash/default.nix

```diff
@@ -20,9 +20,9 @@ rustPlatform.buildRustPackage.override { inherit stdenv; } rec {
       --replace "linker = \"aarch64-linux-gnu-gcc\"" ""
   '';
 
-  cargoHash = "sha256-6uhtOaBsgMw59Dy6yivZYUEWDsYfpInA7VmJrqxDS/4=";
+  cargoHash = "sha256-/KBgUrvxfGlbY+9XVYASwYZUxbhW5q43NTBsvUZSkuo=";
 
-  nativeBuildInputs = [ autoreconfHook cargo cxx-rs hexdump makeWrapper pkg-config ];
+  nativeBuildInputs = [ autoreconfHook cargo cxx-rs git hexdump makeWrapper pkg-config tl-expected ];
```

Comment:
tl-expected looks like it goes to buildInputs.

Reply:
`tl-expected` only contains a single header file. It is not a runtime dependency.

Reply:
I'm going to be away for a while, feel free to make whatever edits you need here. (Maintainer access is on.)

Reply:
I fixed this in the latest PR update.